iree/compiler/Codegen/Interfaces/BufferizationInterfaces.cpp

```diff
@@ -291,6 +407,13 @@ void registerBufferizationInterfaces(DialectRegistry &registry) {
         IREE::Flow::DispatchTensorStoreOp::attachInterface<
             DispatchTensorStoreOpInterface>(*ctx);
       });
+  registry.addExtension(
+      +[](MLIRContext *ctx, IREE::LinalgExt::IREELinalgExtDialect *dialect) {
+        IREE::LinalgExt::ReverseOp::attachInterface<
+            LinalgExtOpInterface<IREE::LinalgExt::ReverseOp>>(*ctx);
+        IREE::LinalgExt::FftOp::attachInterface<
+            LinalgExtOpInterface<IREE::LinalgExt::FftOp>>(*ctx);
```

Comment:
Can this be `IREE::LinalgExt::LinalgExtOp>`?

Reply:
No because `LinalgExtOp` is an interface.

This is what we do in linalg:
```
  registry.addExtension(+[](MLIRContext *ctx, linalg::LinalgDialect *dialect) {
    linalg::InitTensorOp::attachInterface<InitTensorOpInterface>(*ctx);

    // Register all Linalg structured ops. `LinalgOp` is an interface and it is
    // not possible to attach an external interface to an existing interface.
    // Therefore, attach the `BufferizableOpInterface` to all ops one-by-one.
    LinalgOpInterfaceHelper<
#define GET_OP_LIST
#include "mlir/Dialect/Linalg/IR/LinalgStructuredOps.cpp.inc"
        >::registerOpInterface(ctx);
  });
```

We may be able to do something similar here. We can do that in a follow-up commit, which would also add bufferization test cases for each op.

Reply:
Yes, thats the only way to redirect implementation of one interface to another interface in core right now.